app/jobs/itms/sync_all_projects_job.rb

```diff
@@ -0,0 +1,15 @@
+require 'harvester_utils/downloader'
```

Comment:
@ebalgava takto by mohli vyzerat tie 'sync all' joby: Nazbiera to hrefy k objektom, ktore ma zaujimaju a potom len zaradim sync joby pre tie objekty do job queue.